pages/posts/w3c-wot-case-study.md

```diff
@@ -0,0 +1,141 @@
+building blocks that make use of JSON Schema.
+
+JSON Schema is used for validating descriptions of network-facing capabilities
+of physical entities called [Thing Descriptions][thing-description], and to
+model and describe data sent by Internet of Things consumers and producers in a
+multi-protocol manner.
+
+The W3C Web of Things specifications are on board with JSON Schema since Draft
+4, and even the first draft versions already catered to data models of Internet
+of Things devices.  "Currently, we are on JSON Schema Draft 7 and expect to
+move to JSON Schema
+[2020-12](https://json-schema.org/specification-links.html#understanding-draft-names-and-numbers)
+or a newer one when starting our new charter in February 2023," said Ege
+Korkan, Research Scientist at Siemens and W3C Specifications editor.
+
+"We observe JSON Schema becoming more and more stable over the years and
+Postman's support to the community gives us even more confidence on it,"
+continued Ege Korkan.
+
+JSON Schema is delivering exciting features, and more importantly these
+features are becoming more and more consistent across implementations.  For
+example, Ege Korkan added, "vocabularies are very promising and we plan to
+explore them further in conjunction with Semantic Web technologies."
+
+## Impact
+
+The W3C Web of Things relies on JSON-LD for linking vocabularies and ontologies
+that already exist on the web, such as [SAREF][saref], [Units of
+Measure][units-of-measure] and [Schema.org][schema-org]. Being based on the
+JSON data model, JSON Schema fits well with JSON-LD when integrating both
+technologies as part of a single specification.
+
+Adopting JSON Schema means that the W3C Web of Things does not need to invest
+effort on inventing another schema language. As a consequence of the popularity
+of JSON Schema, existing tooling can be often reused instead of implementing
+custom parsers, validators, code generators and UI generators for all the
+programming languages that the community might be interested in.
+
+JSON Schema has proven to be a foundational block for creating higher-level
+tooling that is specific to the W3C Web of Things. As a notable example, the
+community has developed a tool that makes use of JSON Schema definitions inside
+Thing Descriptions to generate matching payloads for the purpose of stress and
+penetration testing.
+
+Given the growing popularity of using JSON Schema to generate web-based forms,
+W3C Web of Things specifications can be used to automatically generate
+dashboards and user interfaces to interact with the objects modeled by Thing
+Descriptions.
+
+## Key Impact Results
+
+The ultimate metric of success for a standard like the W3C Web of Things is
+adoption, and community adoption is not something you can just "engineer".  Ege
+Korkan commented, "because we adopt JSON Schema, developers who are familiar
+with it have a smaller learning curve when learning the W3C Web of Things
+standard."
```

Comment:
Hey @egekorkan. Here I'm ideally looking at some kind of soundbite.
Would you be able to take some sort of educated guess as to how much time it saves people getting to grips with the IoT standard?

Reply:
I actually have some sort of numbers on this. During my PhD times, I had quite a bit of students (electrical engineering and masters level) and they had to get up to speed with WoT. A good student took 1 week full time to go through my basic tutorial at https://github.com/tum-esi/wot-tutorial . I think that 1 week is too much for many people since my tutorial goes through Linux and git basics, then some JSON Schema and Node.js before starting with WoT at all. If someone is already somewhat experienced in JSON Schema and some basic Javascript, it should be 2 days full time.

Reply:
Nice stuff! I tried to incorporate that example here. Let me know what you think!